tests/test_ftpfs.py

```diff
@@ -265,7 +265,8 @@ def test_getinfo_modified(self):
         self.assertIn("MDTM", self.fs.features)
         self.fs.create("bar")
         mtime_detail = self.fs.getinfo("bar", ("basic", "details")).modified
-        mtime_modified = self.fs.getinfo("bar", ("modified",)).modified
+        mtime_modified = self.fs.getmodified("bar")
+        print(mtime_detail, mtime_modified)
```

Comment:
Is this `print` supposed to stay or be removed?

Reply:
Ahah, good catch, thanks!